runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/ParDoTranslation.java

```diff
@@ -448,8 +531,37 @@ private static SideInput toProto(PCollectionView<?> view) {
   }
 
   public static PCollectionView<?> fromProto(
```

Comment:
Because this is part of `ParDoTranslation`, and this is public, it may be worth renaming to `viewFromProto` or something similar.

Certainly I believe it would make it easier to review.